modules/webbrowser/webbrowsermodule.cpp

```diff
@@ -225,7 +215,7 @@ namespace webbrowser {
  * described in more detail in globalscallbacks.h.
  */
 
-std::chrono::microseconds interval = std::chrono::microseconds(1);
+std::chrono::microseconds interval = std::chrono::microseconds(10000);
```

Comment:
Maybe a comment why this interval number was chosen. Has it been optimized somehow or is it an estimated value that may be possible to improve further?

Reply:
I added a comment that explains the magic number a bit better. It is recommended from CEF side to call that function "regularly" which is quite unspecific but from testing i feel it doesnt need more then 100fps. I wouldnt go less (higher number) since the gui will not be very responsive again